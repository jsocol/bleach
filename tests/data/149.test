<div id="78"><?xml-stylesheet type="text/xsl" href="#"?><img xmlns="x-schema:test.xdr"/>//["'`-->]]>]</div>
