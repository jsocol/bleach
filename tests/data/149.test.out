<div id="78"><img xmlns="x-schema:test.xdr">//["'`--&gt;]]&gt;]</div>
